src/App.js

```diff
@@ -1,25 +1,49 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import React from "react";
+import NavBar from "./components/navigation/NavBar";
+import {Route, Switch} from "react-router-dom";
+import Home from "./pages/home/Home";
+import Subreddit from "./pages/subreddit/Subreddit";
+// import axios from "axios";
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    // const [redditData, setRedditData] = useState("");
+    //
+    //
+    // useEffect(() => {
+    //     async function fetchData() {
+    //         try {
+    //             const result = await axios.get(`https://www.reddit.com/hot.json?limit=15`);
+    //             console.log(result.data.data.children);
+    //             setRedditData(result.data.data.children);
+    //
+    //         } catch (e) {
+    //             console.error(e);
+    //         }
+    //
+    //     }
+    //
+    //     fetchData();
+    // }, []);
+
+    return (
+        <>
+            <div className="outer_container">
+                <NavBar/>
+
+                <Switch>
+                    <Route exact path="/">
+                        <Home/>
+                    </Route>
+                    <Route path="/subreddit/:subredditId">
+                        <Subreddit/>
+                    </Route>
+                </Switch>
+
+            </div>
+        </>
```

Comment:
Deze lege tags zijn nie tnodig, omdat het maar één child bevat
```suggestion
        <div className="outer_container">
            <NavBar/>
            <Switch>
                <Route exact path="/">
                    <Home/>
                </Route>
                <Route path="/subreddit/:subredditId">
                    <Subreddit/>
                </Route>
            </Switch>
        </div>
```